gufe/chemicalsystem.py

```diff
@@ -29,6 +31,8 @@ class ChemicalSystem(Serializable, abc.Mapping):
 
     """
 
+    _storage_path = "setup/systems/{md5}.json"
+
```

Comment:
Can we instead store this on the storage side of things? So storage has a dict of `{Component: _storage_path}`

Reply:
That would mean that external packages need to also modify our storage when they import their package. I'd rather have everything contained in their class.